Sources/EduIDExpansion/Controllers/Onboarding/HomeViewController.swift

```diff
@@ -115,7 +114,7 @@ class SuccessViewController: EduIDBaseViewController {
     
     @objc
     func personalInfoTapped() {
-        
+        (coordinator as? MainCoordinator)?.showPersonalInfo()
```

Comment:
A view controller should never know about its coordinator. It talks to the coordinator via the delegate pattern. So here you'd have a `HomeViewControllerDelegate` on which a method is define, say `func homeViewController(controller: HomeViewController, showPersonalInfoAnimated: Bool)`. The `HomeViewControllerDelegate` would be implemented in the coordinator and the coordinator would set itself as the delegate of the `HomeViewController` when it creates it.

Reply:
we discussed this

Reply:
I'm open to revisiting it later